src/app/components/forms/form/form-entry/terra-form-entry.component.ts

```diff
@@ -138,7 +138,18 @@ export class TerraFormEntryComponent implements OnInit, AfterViewInit, OnChanges
                     }
                     else
                     {
-                        this.componentInstance[this.transformInputPropertyName(optionKey)] = this.inputFormField.options[optionKey];
+                        if(this.componentInstance.hasOwnProperty(optionKey))
```

Comment:
Ich würde die Abfrage umdrehen:

```
if ( prefixed key exists )
{
  assign prefixed property
}
else if ( property exists )
{
  assign property
}
else
{
  report warning
}
```

Möglicherweise gibt es Komponenten, die beide sowohl eine Eigenschaft mit Prefix, als auch eine ohne haben. In diesem Fall würdest du den wert einer nicht-input-property zuweisen.